woocommerce/woocommerce-admin
# Add height auto on autocomplete popover button

Fixes #7030 

Add height auto on autocomplete popover button

### Screenshots
Before
<img width="327" alt="螢幕快照 2021-06-22 下午7 48 05" src="https://user-images.githubusercontent.com/56378160/123014864-1c1a3700-d395-11eb-9b79-c3b316ef13b9.png">

After
<img width="320" alt="螢幕快照 2021-06-22 下午7 53 29" src="https://user-images.githubusercontent.com/56378160/123014881-22101800-d395-11eb-93b7-37f73a94ca2f.png">


### Detailed test instructions:

- Add products with long titles (make sure they have at least one word in common)
- Save products
- Visit Analytics > Products page to search under: Show: Single Product using the common word.
- See popover buttons render as expected.
